src/rocksdb.rs

```diff
@@ -1952,6 +1962,24 @@ impl DB {
         }
     }
 
+    /// Provide the integrated sst file name that includes the ".sst" ending.
+    pub fn get_range_from_live_file_name(&self, name: &str) -> Option<(Vec<u8>, Vec<u8>)> {
```

Comment:
Since we are exposing `LiveFiles`, why not return live files to TiKV?

Reply:
originally thought that `LiveFiles` relying on index is not suitable for expose. addressed